File: chaeryeon823/PGS_87390.js
// n^2 배열 자르기
// lv 2

function solution(n, left, right) {
  const answer = [];

  for (let i = left; i <= right; i++) {
    const temp = Math.max(Math.floor(i / n), i % n);
    answer.push(temp + 1);
  }

  return answer;
}

File: julia8024/PGS_87390.py
# 프로그래머스
# Lv2 n^2 배열 자르기

def solution(n, left, right):
    answer = []
    for i in range(left, right + 1):
        a = i//n # 몫
        b = i%n # 나머지
        # 무조건 a가 크도록
        if a < b:
            a, b = b, a
        answer.append(a+1)
    return answer
